integration-libs/opf/base/root/services/opf-resource-loader.service.ts

```diff
@@ -91,14 +91,21 @@ export class OpfResourceLoaderService extends ScriptLoader {
     resources: OpfDynamicScriptResource[],
     resolve: (value: void | PromiseLike<void>) => void
   ) {
+    let attributes: any = {
```

Comment:
Sonar warning was raised to use const
detials in https://sonar.tools.sap/project/issues?resolved=false&severities=MINOR&inNewCodePeriod=true&pullRequest=18512&id=composable-storefront

Reply:
Thanks. Solved it.